src/main/scala/beam/router/BeamRouter.scala

```diff
@@ -126,10 +131,29 @@ class BeamRouter(
 
   private var traveTimeOpt: Option[TravelTime] = None
 
+  val kryoSerializer = new KryoSerializer(context.system.asInstanceOf[ExtendedActorSystem])
+
+  private val updateTravelTimeTimeout: Timeout = Timeout(3, TimeUnit.MINUTES)
+
   override def receive: PartialFunction[Any, Unit] = {
     case `tick` =>
       if (isWorkAndNoAvailableWorkers) notifyWorkersOfAvailableWork()
       logExcessiveOutstandingWorkAndClearIfEnabledAndOver
+    case t: TryToSerialize =>
+      val byteArray = kryoSerializer.toBinary(t)
+      log.info("TryToSerialize size in bytes: {}, MBytes: {}", byteArray.size, byteArray.size.toDouble / 1024 / 1024)
+    case UpdateTravelTime_v2(map) =>
+      localNodes.foreach { actor =>
+        log.info("Sending UpdateTravelTime_v2 to  {}", actor)
+        actor.ask(UpdateTravelTime_v2(map))(updateTravelTimeTimeout)
```

Comment:
Should these be `ask`ed? Maybe they should be `tell`ed and have the error listener (or replayer*) handle making sure they are sent?

*Those two classes need merged really